src/packages/@ncigdc/modern_components/Lolliplot/GeneLolliplot.relay.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+/* eslint fp/no-class:0 */
+
+import React from 'react';
+import { graphql } from 'react-relay';
+import { compose, withPropsOnChange } from 'recompose';
+import { makeFilter } from '@ncigdc/utils/filters';
+import Query from '@ncigdc/modern_components/Query';
+
+export default (Component: Object) =>
+  compose(
+    withPropsOnChange(['geneId'], ({ geneId = 'ENSG00000134086' }) => {
```

Comment:
maybe have that as a convention across all modern_components so that they can load by themselves in the `/components/` route